change/@fluentui-react-experiments-2021-02-02-09-54-29-perffixes.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
+  "comment": "Memoize callbacks to reduce UPP re-renders",
+  "packageName": "@fluentui/react-experiments",
+  "email": "elvonspa@microsoft.com",
+  "dependentChangeType": "patch",
```

Comment:
what does it mean that it is a patch? When will this get integrated into OWA? Just want to know to keep an eye on new bug reports in case they are related.